rustler/src/types/local_pid.rs

```diff
@@ -11,6 +11,13 @@ impl LocalPid {
     pub fn as_c_arg(&self) -> &ErlNifPid {
         &self.c
     }
+
+    ///
+    /// Look up a local process by its registered name.
+    ///
+    pub fn whereis<'a>(env: Env<'a>, name: Atom) -> Option<Self> {
```

Comment:
This might as well be a function on `Env` instead. I am not sure what we should prefer here.

```rust
env.whereis(name)
```

vs

```rust
LocalPid::whereis(env, name)
```